python/rbac/endpoints/policy/serializers.py

```diff
@@ -164,7 +165,11 @@ def get_role(obj: Policy) -> str:
     @staticmethod
     def get_object(obj: Policy) -> list[dict[str, int | str]]:
         return [
-            {"id": obj.object.pk, "name": obj.object.name, "type": obj.content_type.name}
+            {
+                "id": obj.object.pk,
+                "name": obj.object.name,
+                "type": get_obj_type(obj.content_type.name),
```

Comment:
Application of this function does not violate the independence of the application from the audit?

Reply:
Refactored